client/status.v2/status_interactive.go

```diff
@@ -191,14 +205,15 @@ func interactive(c Client) error {
 				})
 			}
 		}()
-		t := time.NewTicker(300 * time.Millisecond)
-		for range t.C {
+		for {
 			st, err := c.Status()
 			viewmodelupdate(func(p *viewmodel.Params) {
 				p.Report = st.Jobs
 				p.ReportFetchError = err
 			})
 			app.QueueUpdateDraw(redraw)
+
+			time.Sleep(time.Duration(flag.Delay) * time.Second)
```

Comment:
Main reason why I changed this was because of the ticker used at the beginning of the loop, in case of a delay of 3 seconds you had to wait 3 seconds before getting any response from the UI, that is why I removed the ticker and moved the delay at the end of the loop. I chose the sleep, not sure if it can be done differently.